master/internal/api_tasks.go

```diff
@@ -247,14 +247,54 @@ func (a *apiServer) GetTaskAcceleratorData(
 	ctx context.Context,
 	req *apiv1.GetTaskAcceleratorDataRequest,
 ) (*apiv1.GetTaskAcceleratorDataResponse, error) {
-	return nil, grpcutil.UnimplementedError
+	if req.TaskId == "" {
+		return nil, status.Error(codes.InvalidArgument, "task ID missing")
+	}
+	var res []model.AcceleratorData
+	err := db.Bun().NewSelect().
+		ColumnExpr("alloc_acc.*").
+		TableExpr("allocation_accelerators alloc_acc").
+		Join("LEFT JOIN allocations alloc ON alloc_acc.allocation_id = alloc.allocation_id").
+		Where("alloc.task_id = ?", req.TaskId).Scan(ctx, &res)
+	if err != nil {
+		return nil, fmt.Errorf("querying allocation accelerators: %w", err)
+	}
+	var accelerationData []*apiv1.AcceleratorData
```

Comment:
nit, style, here and elsewhere leave some space between blocks
```suggestion
	}

	var accelerationData []*apiv1.AcceleratorData
```

Reply:
Done